components/tasks/task-list.tsx

```
@@ -0,0 +1,199 @@
+    const matchesStatus = statusFilter === 'all' || task.status === statusFilter
+    const matchesType = typeFilter === 'all' || task.type === typeFilter
+    const matchesPriority = priorityFilter === 'all' || task.priority === priorityFilter
+
+    return matchesSearch && matchesStatus && matchesType && matchesPriority
+  })
+
+  // Sort tasks
+  const sortedTasks = [...filteredTasks].sort((a, b) => {
+    switch (sortBy) {
+      case 'priority':
+        const priorityOrder = { HIGH: 0, MEDIUM: 1, LOW: 2 }
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)